examples/amp-story/access.html

```diff
@@ -0,0 +1,83 @@
+<!doctype html>
+<html amp lang="en">
+  <head>
+    <meta charset="utf-8">
+    <script async src="https://cdn.ampproject.org/v0.js"></script>
+    <script async custom-element="amp-story"
+        src="https://cdn.ampproject.org/v0/amp-story-1.0.js"></script>
+    <script async custom-element="amp-access"
```

Comment:
Maybe we should think about making this a separate extension?  We do want to make sure we keep the amp-story binary size down (and should probably do work in the near future to push for that)

Reply:
Sounds good, I scheduled some time tomorrow so we can talk about it.

My biggest worry is that having to require ``amp-story`` + ``amp-access`` + ``amp-story-access`` can get confusing, especially for us, in terms of compatibility with both `0.1` and `1.0`.